setup.cfg

```diff
@@ -1,4 +1,4 @@
 [bdist_rpm]
```

Comment:
We use fpm to generate the RPM and I don't think `bdist_rpm` is used anymore.
I'd remove that `setup.cfg` file.